single_stage_detector/download_dataset.sh

```diff
@@ -1,7 +0,0 @@
-# Get COCO 2017 data sets
```

Comment:
we also need to update the README of the `single_stage_detector/` benchmark in order for this change to be reflected